src/SOS/SOS.NETCore/SymbolReader.cs

```diff
@@ -211,24 +214,21 @@ public static bool InitializeSymbolStore(bool logging, bool msdl, bool symweb, s
         /// <summary>
         /// Displays the symbol server and cache configuration
         /// </summary>
-        public static void DisplaySymbolStore()
+        public static void DisplaySymbolStore(WriteLine writeLine)
```

Comment:
Is there a reason why this one logs to a delegate and the TraceListener logs to the IConsoleService?

Reply:
Tracing may not be enabled or pointing to the console and this function is used by the "setsymbolserver" command to display the symbol stores that are configured. The delegate passed is basically the native "ExtOut" SOS function.

Reply:
Isn't IConsoleService also a wrapper for the ExtOut function?